app/storybook/documentation/ui-kit/api/MetricInput.mdx

```diff
@@ -0,0 +1,33 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
```

Comment:
This also looks wrong.
<img width="985" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/3609104d-9d2a-4afc-ade3-0a9d05929b23">

We should show type names instead of dashes, e.g. for `average` description should be:
```
An ad hoc Average Metric.
[AverageMetricQueryInput](http://localhost:6006/?path=/story/api-averagemetricqueryinput--overview)
```

Like we do in other places
<img width="958" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/b18eca62-8df9-4e5f-970d-f75d9961f047">

I would also suggest keeping the API pages simple, for example, by omitting the references list at the bottom. Since the links are hardcoded, there's a risk of them becoming outdated quickly. It's not a problem to manually find a proper API page (at least we provide them 😄 ).